### PR TITLE
hotfix remove comma

### LINKED ISSUE #124: Parse error caused by a comma
### Describe the bug

Reported in the support - https://wordpress.org/support/topic/parse-error-critical-with-update-1-2-0/

### Steps to Reproduce

I am unable to reproduce the issue, this might be reported on a specific version of PHP

### Screenshots, screen recording, code snippet

_No response_

### Environment information

_No response_

### WordPress information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Sidsector9**
### Description of the Change

The release of 1.2.0 introduced a parse error as reported here - https://wordpress.org/support/topic/parse-error-critical-with-update-1-2-0/

This issue is not reproducible, I am guessing it is reported on a different version of PHP.

Closes #124 

### Changelog Entry

> Fixed - Parse error caused by a comma

### Credits

Props @Sidsector9 

### Checklist:

- [ ] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Review by iamdharmesh — COMMENTED** [review]
Thanks for working on this and fixing this. I am not able to reproduce this but changes looks good to me.

However, based on error it looks like issue reporter may using PHP < 7.3 as support for trailing comma  added in [PHP 7.3](https://wiki.php.net/rfc/trailing-comma-function-calls). This plugin requires minimum PHP 8. So, not sure if should release this in Hotfix release. It's fine to release this with our next scheduled release instead of hotfix release for only this issue.

@dkotter @jeffpaul any thoughts here?

Thanks

**Comment by jeffpaul** [comment]
Perhaps including #122 will help ensure folks are on >=PHP8?

**Comment by dkotter** [comment]
I don't think we should have a trailing comma here in the first place so I'm fine with getting this fix out.

That said, this is now the second time we've had someone be able to update one of our plugins when their PHP version doesn't match our required minimum (Safe SVG being the other). It seems the minimum PHP requirement in WordPress doesn't always work, whether that's a core issue or some conflict with various security/backup type plugins.

The way we addressed that in Safe SVG was by adding our own PHP check (see https://github.com/10up/safe-svg/blob/develop/safe-svg.php#L54). We've done the same in Distributor as well. I'd suggest we start adding this same check to all of our plugins so if someone is able to update/install when they shouldn't be able to, we are able to catch that and show a warning, rather than cause PHP errors. I would suggest we add a similar check as part of #122.

**Comment by jeffpaul** [comment]
Yeah, that unfortunately sounds like the safest path @dkotter.  @vikrampm1 mind adding issues across our WP plugin repos to add our own PHP check and then move those all to To Do so we can try and get that work covered across the 10up team?

**Comment by ravinderk** [comment]
> Yeah, that unfortunately sounds like the safest path @dkotter. @vikrampm1 mind adding issues across our WP plugin repos to add our own PHP check and then move those all to To Do so we can try and get that work covered across the 10up team?

@vikrampm1, did you open issues?

**Comment by vikrampm1** [comment]
@ravinderk thanks for the ping. I haven't yet, I will update back here once they are opened.

**Comment by vikrampm1** [comment]
@jeffpaul @dkotter I have opened the issues across all the WP plugin repos and added them to our to-do column as high priority, let me know if I missed anything. cc @ravinderk

**Review by ravinderk — DISMISSED** [review]

**Comment by ravinderk** [comment]
@dkotter Can we release this hotfix?

File: includes/ConvertToBlocks/Settings.php [hunk 1]
```diff
@@ -256,7 +256,7 @@ public function filter_notice() {
 			<p>
 				<?php
 				printf(
-					esc_html__( 'A filter hook (post_type_supports_convert_to_blocks) is already active.', 'convert-to-blocks' ),
+					esc_html__( 'A filter hook (post_type_supports_convert_to_blocks) is already active.', 'convert-to-blocks' )
 				);
 				?>
```
Comment (felipeelia) [inline includes/ConvertToBlocks/Settings.php]: This whole block should actually become
```php
<?php esc_html_e( 'A filter hook (post_type_supports_convert_to_blocks) is already active.', 'convert-to-blocks' ) ?>
```
no?

**Review by dkotter — DISMISSED** [review]

**Comment by Sidsector9** [comment]
@dkotter instead of implementing the PHP in each repository, can we maintain our own composer "utility" package common for all plugins?

As for conflicts between 2 or more plugins using the same package, we can use [php-scoper](https://github.com/humbug/php-scoper) to avoid conflitcs?

I tested out a small package I made - https://github.com/Sidsector9/wp-utils. We can move this to a 10up repository and use it across all repos.

**Pull request merged**